docs/release_process.rst

```diff
@@ -0,0 +1,195 @@
+=======================
+CCXCon Release Workflow
+=======================
+
+Quick Start
+-----------
+
+prerequisites:
+  - ``git-release-notes`` - https://www.npmjs.com/package/git-release-notes
+  - ``git-flow`` - https://github.com/nvie/gitflow
+  - You've configured Git-Flow to use ccxcon branch naming scheme.
+    (See Git-Flow_ section below)
+
+These instructions presume you're releasing ccxcon version 0.10.0.
+
+.. code-block:: bash
+
+    git fetch
+    git checkout master
+    git pull
+    git flow release start 0.10.0
+    vi ccxcon/settings.py # change version number
+    git-release-notes v0.9.0..master util/release_notes_rst.ejs
+    vi RELEASE.rst # paste results of prior command at top of file, updating the release number
+    git add RELEASE.rst ccxcon/settings.py
+    git commit -m "Release 0.10.0"
+    git flow release publish 0.10.0
+    # Verify that the Travis-ci build succeeds
+    git-release-notes v0.9.0..master util/release_notes.ejs
```

Comment:
These are two different scripts, though you might miss that at first glance.  One creates a reStructuredText document that is included in our ReadTheDocs documentation, and the other creates checkboxes for the ``release`` PR description.